app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -18,7 +18,11 @@ module HostManagedExtensions
         has_many :host_installed_packages, :class_name => "::Katello::HostInstalledPackage", :foreign_key => :host_id, :dependent => :destroy
         has_many :installed_packages, :class_name => "::Katello::InstalledPackage", :through => :host_installed_packages
 
+        has_many :host_collection_hosts, :class_name => "::Katello::HostCollectionHosts", :foreign_key => :host_id, :dependent => :destroy
+        has_many :host_collections, :class_name => "::Katello::HostCollection", :through => :host_collection_hosts, :foreign_key => :host_collection_id
+
         scoped_search :in => :content_source, :on => :name, :complete_value => true, :rename => :content_source
+        scoped_search :in => :host_collections, :on => :id, :complete_value => false, :rename => :host_collection_id
```

Comment:
can you add a search for host collection name?  I'd probably expect it to be just 'host_collection = foo'